test-pom/pom.xml

```diff
@@ -317,9 +317,16 @@ THE SOFTWARE.
     <profile>
       <!-- Profile, which runs only a number of quick integration tests so that the build takes less than 5 minutes -->
       <id>smoke-test</id>
-      <properties>
-        <test>jenkins.model.StartupTest,jenkins.model.JenkinsTest,jenkins.install.*,jenkins.security.CustomClassFilterTest,hudson.AboutJenkinsTest,hudson.ClassicPluginStrategyTest,hudson.LauncherTest,hudson.slaves.JNLPLauncherTest,hudson.model.RunTest,hudson.model.UserTest,hudson.model.FreeStyleProjectTest,hudson.model.HudsonTest,hudson.model.ComputerTest,hudson.CLI.BuildCommandTest#sync,*.smokes</test>
-      </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <groups>org.jvnet.hudson.test.SmokeTest</groups>
```

Comment:
will it properly append the configuration in such case?

Reply:
@oleg-nenashev Yes, the tests are being carried out at the end of the build. You may verify from [this](https://pastebin.com/kuwgqNnr) build log.